cfme/intelligence/reports/reports.py

```diff
@@ -330,7 +330,7 @@ def update(self, updates):
             view.save_button.click()
         else:
             view.cancel_button.click()
-        view = self.create_view(ReportDetailsView, override=updates, wait="5s")
+        view = self.create_view(ReportDetailsView, override=updates, wait="60s")
```

Comment:
@valaparthvi Why does increasing the `wait` time allow your test to pass?

Reply:
I was getting a `NoSuchElementException` for `Report Info` tab which appears after the report is edited. Giving it some time, helped in avoiding the error.

Reply:
@valaparthvi What if you did a browser refresh after creating the view instead waiting 60s, would that solve the issue? Seems like a long wait.

Reply:
Okay, I'll see if browser refresh helps, or if there's some other way.